src/main/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -201,6 +207,10 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
 
         creativeResolve = defineBoolean(builder, "creativeresolve", false);
 
+        swapToCategory(builder, "debugging");
+
+        netherWorkerTakesDamage = defineBoolean(builder, "netherworkertakesdamage", true);
```

Comment:
I wouldn't add that as a config, the nether worker should just take damage. Otherwise people should datapack the netherworker stuff

Reply:
still applies

Reply:
But we can remove it later if it turns out it is not needed.
We have many of these debugging flags, things like creative resolve etc.

So I don't see this as a particular problem.

Reply:
The problem is less about it being needed, the problem is that tomorrow ATM might just turn on this config randomly.